.template/spec/base/config/environments/production_spec.rb

```diff
@@ -9,6 +9,10 @@
     expect(subject).to contain(mailer_default_url_config)
   end
 
+  it 'allow Rails serve static file' do
+    expect(subject).to contain('config.public_file_server.enabled = true')
```

Comment:
Disabled it in Nginx addon PR https://github.com/nimblehq/rails-templates/pull/232/files#diff-c7e70c5ce93b28441e8a240fa5a19594R8